d2common/d2enum/inventory_item_type.go

```diff
@@ -3,7 +3,7 @@ package d2enum
 type InventoryItemType int
 
 const (
-	InventoryItemTypeItem   InventoryItemType = 0 // Item
+	InventoryItemTypeItem   InventoryItemType = 0 // item
```

Comment:
Why?

Reply:
I believe this was added after I did a `git pull` last time, I fixed it now.